common/tqid/metrics_helper.go

```diff
@@ -0,0 +1,90 @@
+// The MIT License
+//
+// Copyright (c) 2020 Temporal Technologies Inc.  All rights reserved.
+//
+// Copyright (c) 2020 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package tqid
+
+import (
+	"strconv"
+
+	"go.temporal.io/server/common/metrics"
+)
+
+const (
+	unknown = "_unknown_"
+	omitted = "__omitted__"
+	normal  = "__normal__"
+	sticky  = "__sticky__"
+)
+
+func GetPerTaskQueueFamilyScope(
+	handler metrics.Handler,
+	namespaceName string,
+	taskQueueFamily *TaskQueueFamily,
+	taskQueueBreakdown bool,
+) metrics.Handler {
+	metricTaskQueueName := omitted
+	if taskQueueBreakdown {
+		metricTaskQueueName = taskQueueFamily.Name()
+	}
+	return handler.WithTags(
+		metrics.NamespaceTag(namespaceName),
+		metrics.UnsafeTaskQueueTag(metricTaskQueueName),
```

Comment:
it looks like this is true but just checking: we always emit the same tag set and just change the values, right?